src/javascript/editorTabs/EditPanelContent/FormBuilder/Field/Field.jsx

```diff
@@ -88,6 +88,8 @@ export const Field = ({inputContext, idInput, selectorType, field}) => {
 
     const initialValue = useRef(values[field.name]);
 
+    const currentValue = useMemo(() => values[field.name], [values, field.name]);
```

Comment:
Why do we need a memo here ? Can't we use `values[field.name]` directly in the effect ?